python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -652,6 +651,56 @@ def train_step(self, graph, feat, tmp, **kwargs):
         """
         return super().train_step(graph, feat, tmp=tmp, **kwargs)
 
+    def train_step_node(self, nodes, graph, feat, tmp, **kwargs):
```

Comment:
```suggestion
    def train_step_node(self, nodes, graph, feat, temperature, **kwargs):
```
`tmp` has too many meanings; it can also mean "temporary" etc.